src/u2f/u2f_app.c

```diff
@@ -139,6 +161,16 @@ static void _app_string(const uint8_t* app_id, char* out, size_t out_len)
             return;
         }
     }
+    char* mnemonic=NULL;
+    //fallback to hex if something goes wrong
+    if (bip39_mnemonic_from_bytes(NULL, app_id, 32, &mnemonic) == WALLY_OK) {
```

Comment:
this uses `malloc` to allocate `mnemonic`, but a `free(mnemonic)` is missing, leading to a memory leak.

Reply:
done